articles/virtual-machines/workloads/redhat/redhat-rhui.md

```diff
@@ -203,6 +203,7 @@ RHUI 3
 52.174.163.213
```

Comment:
Can we add the region name here also as we have in RHUI4 to ensure uniformity?

Reply:
we shouldn't add region name as folks may add a selected region. For RHUI 4 this information is already published, hence not removing it, and have added comments before the RHUi 4 block